libs/ngx-charts-on-fhir/src/lib/data-layer-list/dataset-options/dataset-options.component.ts

```diff
@@ -38,6 +40,7 @@ export class DatasetOptionsComponent implements OnInit {
     this.form.valueChanges.subscribe((value) => {
       this.updateModel(value);
     });
+    this.showAdvancedOptions = this.sharedDataService.getOptionPanelValue();
```

Comment:
why is that service using a subject if the component is only retrieving the value once on init? Shouldn't the component be subscribing instead so UI will update whenever the option changes?